### PR TITLE
Create basic deductible selector

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Create basic `DeductibleSelector` component. Based on `TierSelector` component.

## Justify why they are needed

Will later be added to `OfferSelector` to allow user to select deductible. But we need the API first.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1901** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1901" target="_blank">Graphite</a> 
    * **PR #1902** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1902" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1902?utm_source=stack-comment).

File: apps/store/src/utils/PageLink.ts [hunk 1]
```diff
@@ -62,3 +71,8 @@ const CUSTOMER_SERVICE_URL: Partial<Record<RoutingLocale, string>> = {
   se: '/se/hjalp/kundservice',
   'en-se': '/se-en/help/customer-service',
 }
+
+const DEDUCTIBLE_HELP_URL: Partial<Record<RoutingLocale, string>> = {
```
Comment (alebedev) [inline apps/store/src/utils/PageLink.ts:75]: Just a background thought, we're relying on specific CMS paths in code more and more.  There should be some standard way to work with those

Comment (robinandeer) [reply apps/store/src/utils/PageLink.ts:75]: Yeah for sure!

We should probably move this to some global CMS config in Storyblok 🙂

**Comment by robinandeer** [comment]
@robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1902).

**Pull request merged**